requirements.txt

```diff
@@ -22,6 +21,9 @@ watchfiles==0.15.0
 #
 git+https://github.com/aiven/kafka-python.git@1b95333c9628152066fb8b1092de9da0433401fd
 
+git+https://github.com/aiven/avro.git@branch-1.11#subdirectory=lang/py
```

Comment:
Everything else looks good, but IMHO this should point to git sha, not branch name (as branch can evolve) ie. replacing `branch-1.11` here with current branch head sha1sum. `branch-1.11` can be used as a comment here explain why this optionally. What do you think?